eq3bt/eq3btsmart.py

```diff
@@ -388,5 +403,11 @@ def max_temp(self):
         return EQ3BT_MAX_TEMP
 
     @property
-    def away_end(self):
-        return self._away_end
```

Comment:
Was this removal accidental?

Reply:
No, it was intentional. There is a completely identical declaration on line 236-238. Therefore my IDE flagged this as a duplicate.